src/modules/markets/markets.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { MarketEntity } from './entities/market.entity';
+import { DbResponseHelper } from '../../helpers/db-response.helper';
+
+@Injectable()
+export class MarketsService {
+  constructor(
+    @InjectRepository(MarketEntity, 'mssql-cis')
+    private readonly marketsRepository: Repository<MarketEntity>,
+  ) {}
+
+  async findAll(active?: string): Promise<MarketEntity[]> {
+    let spResults = await this.marketsRepository.query('CIS_USP_READ_MARKETS');
+
+    if (active !== undefined) {
```

Comment:
we could use here: `if (active) {...}`
this will check if there is a value - it checks if the value is `null` as well

Reply:
Changed to shorter version